zerver/lib/email_mirror.py

```diff
@@ -210,6 +212,27 @@ def send_zulip(sender: UserProfile, stream: Stream, topic: str, content: str) ->
     )
 
 
+def send_mm_reply_to_stream(
+    user_profile: UserProfile, stream: Stream, topic: str, body: str
+) -> None:
+    try:
+        check_send_message(
+            sender=user_profile,
+            client=get_client("Internal"),
+            message_type_name="stream",
+            message_to=[stream.id],
+            topic_name=topic,
+            message_content=body,
+        )
+    except JsonableError as error:
+        error_message = "Error sending message to stream {stream}:\n{error}".format(
+            stream=stream.name, error=error.msg
+        )
+        internal_send_private_message(
+            get_system_bot(settings.ERROR_BOT), user_profile, error_message
```

Comment:
We should use NOTIFICATION_BOT here, error bot is not used for sending messages to users - and I think actually the only reason this works here is because in dev, error bot lives in the `zulip` realm. In production:
1. It may not exist at all
2. If it exists, it still exists in one realm and will not be able to send messages to users from other realms - because unlike NOTIFICATION_BOT, it is not a cross-realm bot.

Reply:
Okay, I will replace it.